packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -322,7 +322,6 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
         // fix the branch name in attributes
         // this prevents issues when generating summaries
         const documentAttributes: api.IDocumentAttributes = JSON.parse(bufferToString(blob, "utf8"));
-        documentAttributes.branch = this.documentId;
         const content = JSON.stringify(documentAttributes);
```

Comment:
you should be able to remove this whole block, as the only reason it exists is to patch branch info